src/errors.rs

```diff
@@ -0,0 +1,36 @@
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum FeelessError {
+    #[error("from hex error")]
```

Comment:
These error strings are reported to the user sometimes, so I prefer them to be capitalized sentences. I'm happy to change it myself because I'm picky and it's a pain. Let me know!